client/state/action-types.js

```diff
@@ -16,6 +16,7 @@ export const EDITOR_CONTACT_FORM_ADD_DEFAULT_FIELD = 'EDITOR_CONTACT_FORM_ADD_DE
 export const EDITOR_CONTACT_FORM_CLEAR_FORM = 'EDITOR_CONTACT_FORM_CLEAR_FORM';
 export const EDITOR_CONTACT_FORM_LOAD_FORM = 'EDITOR_CONTACT_FORM_LOAD_FORM';
 export const EDITOR_CONTACT_FORM_REMOVE_FIELD = 'EDITOR_CONTACT_FORM_REMOVE_FIELD';
+export const EDITOR_POST_CURRENT_ID = 'EDITOR_POST_CURRENT_ID';
```

Comment:
Should probably have some sort of verb in this action type name. Consider suffixing with `_SET`